packages/core/components/FileMetadataSearchBar/FileMetadataSearchBar.module.css

```diff
@@ -0,0 +1,44 @@
+.search-bar-container {
+    border: solid black 1px;
+    border-radius: 3px;
+    display: flex;
+    margin-bottom: 5px;
+}
+
+.filter-input div i, .cancel-button, .cancel-button:hover {
```

Comment:
Here and below: fluentui provides explicit mechanisms for applying custom classnames to very specific subcomponents. Is there a reason why you're not using that mechanism? This is extremely fragile.